src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskStreamingTest.java

```diff
@@ -88,17 +88,17 @@ public void testSinkTaskInvalidRecord_InMemoryDLQ() throws Exception {
         new TopicPartitionChannel(
             mockStreamingClient,
             topicPartition,
-            SnowflakeSinkServiceV2.partitionChannelKey(TEST_CONNECTOR_NAME, topicName, partition),
```

Comment:
Do we have any tests that can actually test the end2end migration?

Reply:
all end to end tests will eventually go through this path right? since it is default to true.

Reply:
good idea to ingest through topicpartition channel through v2 and then let it go through migration.. I will add IT tests..

Reply:
I think I see the issue, the IT tests are showing up other PR: https://github.com/snowflakedb/snowflake-kafka-connector/pull/750/files
I might have messed up with merge conflicts.

Reply:
ported the IT tests here, added few more tests in connectionIT tests.